docs/source/tutorials/token_classification.ipynb

```diff
@@ -0,0 +1,730 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e6e2963",
+   "metadata": {},
+   "source": [
+    "Let's look at the top 20 examples cleanlab thinks are most likely to be incorrectly labeled. We obtain the sentences from the original files to display the word-level token label issues in context. `given_words` contains the word-level tokens in the dataset such that `given_words[i]` is a list of words of the `i`'th sentence; `given_labels` contains the given labels in the dataset such that `given_labels[i]` is a list of labels of the `i`th sentence. Note that in CoNLL-2003, the `B-` and `I-` prefixes indicates whether the tokens are at the start of an entity, which are ignored in this tutorial. Therefore, we have two sets of entities: \n",
+    "\n",
+    "    given_entities = ['O', 'B-MISC', 'I-MISC', 'B-PER', 'I-PER', 'B-ORG', 'I-ORG', 'B-LOC', 'I-LOC'] \n",
+    "     \n",
+    "and \n",
+    "\n",
+    "    merge_entities = ['O', 'MISC', 'PER', 'ORG', 'LOC'] "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ac24b72b",
+   "metadata": {},
+   "source": [
+    "<details><summary>Below is the code used for reading the files.</summary>\n",
+    "\n",
+    "    filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
+    "    given_entities = ['O', 'B-MISC', 'I-MISC', 'B-PER', 'I-PER', 'B-ORG', 'I-ORG', 'B-LOC', 'I-LOC']\n",
+    "    merged_entities = ['O', 'MISC', 'PER', 'ORG', 'LOC'] \n",
+    "    entity_map = {entity: i for i, entity in enumerate(given_entities)} \n",
+    "\n",
+    "    def readfile(filepath, sep=' '): \n",
+    "        lines = open(filepath)\n",
+    "\n",
+    "        data, sentence, label = [], [], []\n",
+    "        for line in lines:\n",
+    "            if len(line) == 0 or line.startswith('-DOCSTART') or line[0] == '\\n':\n",
+    "                if len(sentence) > 0:\n",
+    "                    data.append((sentence, label))\n",
+    "                    sentence, label = [], []\n",
+    "                continue\n",
+    "            splits = line.split(sep) \n",
+    "            word = splits[0]\n",
+    "            if len(word) > 0 and word[0].isalpha() and word.isupper():\n",
+    "                word = word[0] + word[1:].lower()\n",
+    "            sentence.append(word)\n",
+    "            label.append(entity_map[splits[-1][:-1]])\n",
+    "\n",
+    "        if len(sentence) > 0:\n",
+    "            data.append((sentence, label))\n",
+    "\n",
+    "        given_words = [d[0] for d in data] \n",
+    "        given_labels = [d[1] for d in data] \n",
+    "\n",
+    "        return given_words, given_labels \n",
+    "\n",
+    "    given_words, given_labels = [], [] \n",
```

Comment:
We should always use separate cells to define the helper functions vs the code that uses the helper functions